aws/Dockerfile

```diff
@@ -8,8 +8,8 @@ RUN apt-get -yqq update && apt-get -yqq install git-all
 #NOTE: make sure ssh keys are added to ssh_keys folder
 
 RUN mkdir root/tmp_ssh
-COPY /ssh_keys/. /root/.ssh/
-RUN cd /root/.ssh && chmod 600 * && chmod 644 *.pub
+COPY ./ssh_keys/ /root/.ssh/
+RUN chmod 400 -R /root/.ssh
```

Comment:
The correct ssh folder permissions are 700 for ~/.ssh and 600 for files inside that folder. I added [a pull request](#1713) before I saw this one.